beacon-chain/sync/service.go

```diff
@@ -98,7 +98,7 @@ type Checker interface {
 	Status() error
 }
 
-// HelloTracker interface for accessing the hello / handshake messages received so far.
-type HelloTracker interface {
-	Hellos() map[peer.ID]*pb.Hello
+// PeerTracker interface for accessing the status / handshake messages received so far.
+type PeerTracker interface {
```

Comment:
Can we call this `StatusTracker`? 

Also does it need to be public? (still drinking coffee right now so this might be dumb question)

Reply:
it needs to be public as the initial-sync package also uses it